diff --git a/src/app/components/signin/Signin.js b/src/app/components/signin/Signin.js
index 7bcbd2c..30a750b 100644
--- a/src/app/components/signin/Signin.js
+++ b/src/app/components/signin/Signin.js
@@ -116,15 +116,6 @@ class Signin extends Component {
                   Signin
                 </button>
               }
-                <p className="text-center">
-                  <br/><br/>or signin with
-                </p>
-                <a href="/apis/auth/google"
-                        type="submit"
-                        className="btn btn-primary btn-block"
-                        disabled={ loading }>
-                  Google
-                </a>
               </div>
             </form>
           </div>
